scripts/prepare-machine.ps1

```diff
@@ -117,27 +114,9 @@ if ($InitSubmodules) {
     }
 }
 
-function Install-ClogTool {
-    param($ToolName)
-    New-Item -Path $NuGetPath -ItemType Directory -Force | Out-Null
-    $NuGetName = "$ToolName.$ClogVersion.nupkg"
-    $NuGetFile = Join-Path $NuGetPath $NuGetName
-    try {
-        if (!(Test-Path $NuGetFile)) {
-            Write-Host "Downloading $ClogDownloadUrl/$NuGetName"
-            Invoke-WebRequest -Uri "$ClogDownloadUrl/$NuGetName" -OutFile $NuGetFile
-        }
-        Write-Host "Installing: $NuGetName"
-        dotnet tool update --global --add-source $NuGetPath $ToolName
-    } catch {
-        if ($FailOnError) {
-            Write-Error $_
-        }
-        $err = $_
-        $MessagesAtEnd.Add("$ToolName could not be installed. Building with logs will not work")
-        $MessagesAtEnd.Add($err.ToString())
-    }
-}
+Write-Host "Initializing clog submodule"
+git submodule init submodules/clog
+git submodule update
```

Comment:
I'm ok with this for now, but we should probably figure out how to be more selective for this in the future.